### ISSUE TITLE
apply the Apache license

### ISSUE THREAD

**Issue by graingert** [astral-sh/ruff-pre-commit#108]
currently only the instructions on how to apply the apache license, and the license text have been added to the repo. You need to include a copyright statement

**Comment by MichaReiser** [comment]
Thanks for opening the issue. I'm not very familiar with legal. Are you asking for something similar to https://github.com/rust-lang/rust/blob/master/COPYRIGHT ?

**Comment by graingert** [comment]
Yes something like that, per the instructions here: https://github.com/astral-sh/ruff-pre-commit/blob/main/LICENSE-APACHE#L179

**Comment by zanieb** [comment]
My understanding is that https://github.com/astral-sh/ruff-pre-commit?tab=readme-ov-file#license is sufficient
